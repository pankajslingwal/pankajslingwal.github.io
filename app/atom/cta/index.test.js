import React from 'react';
import renderer from 'react-test-renderer';
import CTA from './';


describe("CTA", () => {

    it('renders correctly', () => {
        const tree = renderer.create(<CTA />).toJSON();
        expect(tree).toMatchSnapshot();
    }) 

});