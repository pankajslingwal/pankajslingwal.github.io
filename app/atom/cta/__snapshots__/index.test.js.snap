// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CTA renders correctly 1`] = `
<button
  className=""
  onClick={[Function]}
/>
`;
